word_scripts/wordfind_site.py

```diff
@@ -0,0 +1,26 @@
+import requests
+from bs4 import BeautifulSoup
+
+def wordFindSite ():
+    wordFind_dict = {}
+    for word_length in range(3,12):
+        try:
+            req = requests.get("http://www.wordfind.com/{0}-letter-words/".format(word_length))
+        except:
+            pass
+        
+        unextracted = BeautifulSoup(req.text, 'html.parser').find_all('li')
+        
+        def extract (htmlList):
```

Comment:
Bring this extract function outside of this function's scope (put it into the global scope) so it's easier to read. It's not common to nest functions in functions because it makes it generally hard to read